src/main/java/neqsim/processSimulation/processEquipment/compressor/Compressor.java

```diff
@@ -310,6 +310,16 @@ public double findOutPressure(double hinn, double hout, double polytropicEfficie
   @Override
   public void run() {
     thermoSystem = inStream.getThermoSystem().clone();
+    
+    if(Math.abs(pressure-thermoSystem.getPressure(pressureUnit))<1e-6 && !compressorChart.isUseCompressorChart()) {
```

Comment:
the comparison with 1e-6 smells like a unit specific comparison. PressureUnit is a variable?

Reply:
Thanks - I will try to fix it.